src/lib/swapper/swappers/LifiSwapper/utils/lifiTokenToAssetId/lifiTokenToAssetId.ts

```diff
@@ -0,0 +1,64 @@
+import type { Token } from '@lifi/sdk'
+import type { AssetId, ChainReference } from '@shapeshiftoss/caip'
+import {
+  ASSET_NAMESPACE,
+  ASSET_REFERENCE,
+  CHAIN_NAMESPACE,
+  CHAIN_REFERENCE,
+  toAssetId,
+  toChainId,
+} from '@shapeshiftoss/caip'
+import { SwapError, SwapErrorType } from 'lib/swapper/api'
+
+import { DEFAULT_LIFI_TOKEN_ADDRESS } from '../constants'
+
+export const lifiTokenToAssetId = (lifiToken: Token): AssetId => {
+  const chainId = toChainId({
+    chainNamespace: CHAIN_NAMESPACE.Evm,
+    chainReference: lifiToken.chainId.toString() as ChainReference,
+  })
```

Comment:
Not needed as part of this PR but might want to add a `toLifiChainId` since Li.Fi exclusively deals with EVM chains and I feel like we might need to re-use this

Reply:
agreed but ill hold off on this one for now